### PR TITLE
Deeplearning4j -nn clean up

### PR DESCRIPTION
**Pull request by agibsonccc**
## What changes were proposed in this pull request?
Remove old helpers
Update long indices in module simlar to other int -> long changes.
Misc spacing fixes in deeplearning4j-nn
## How was this patch tested?

Manually
## Quick checklist

The following checklist helps ensure your PR is complete:

- [X ] Eclipse Contributor Agreement signed, and signed commits - see [IP Requirements](https://deeplearning4j.konduit.ai/multi-project/how-to-guides/contribute/eclipse-contributors) page for details
- [X ] Reviewed the [Contributing Guidelines](https://github.com/eclipse/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [ X] Created tests for any significant new code additions.
- [X ] Relevant tests for your changes are passing.

### CODE REVIEW

**Comment by treo** [comment]
This seems to include #10073, I'll do a review of this after it has been merged.

**Comment by treo** [comment]
Update this branch to master as well please.

File: deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/serde/MultiLayerConfigurationDeserializer.java [hunk 1]
```diff
@@ -66,8 +93,9 @@ public MultiLayerConfiguration deserialize(JsonParser jp, DeserializationContext
         boolean requiresLegacyWeightInitHandling = requiresWeightInitFromLegacy(layers);
         boolean requiresLegacyActivationHandling = requiresActivationFromLegacy(layers);
         boolean requiresLegacyLossHandling = requiresLegacyLossHandling(layers);
-
+        ObjectMapper mapper = mapper();
         if(attemptIUpdaterFromLegacy || requiresLegacyRegularizationHandling || requiresLegacyWeightInitHandling) {
+            System.out.println("Legacy mapping");
```
Comment (treo) [inline deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/serde/MultiLayerConfigurationDeserializer.java:98]: Stray debug print?

File: deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/layers/convolution/CnnLossLayer.java [hunk 2]
```diff
@@ -157,6 +157,15 @@ public INDArray activate(boolean training, LayerWorkspaceMgr workspaceMgr) {
...
         INDArray out2d = layerConf().getActivationFn().getActivation(input2d, training);
+        //just  print all inputs and outputs + method name
+        System.out.println("CnnLossLayer activation - forward pass input ("
+                + layerId() + " - " + this.layerConf().getLayerName() + " )");
+        System.out.println("input: " + input.toStringFull());
+        System.out.println("CnnLossLayer activation - forward pass result ("
+                + layerId() + " - " + this.layerConf().getLayerName() + " )");
+        System.out.println("Output shape: " + Arrays.toString(out2d.shape()));
+        System.out.println("Output: " + out2d.toStringFull());
+
```
Comment (treo) [inline deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/layers/convolution/CnnLossLayer.java:168]: Stray debug print?

**Review by treo — CHANGES_REQUESTED** [review]
It looks like there are a couple of stray debug prints left over here.

**Comment by agibsonccc** [comment]
Done.

**Pull request closed without merging**